lib/chance-generators.js

```diff
@@ -580,52 +580,83 @@
       return g
     })
 
-    var lastMagicValues = null
-    var magicStringGenerator = null
-    function getMagicStringGenerator (magicValues) {
-      if (lastMagicValues && lastMagicValues.size === magicValues.size) {
-        return magicStringGenerator
-      }
+    function hasMagicValues () {
+      return global.recordLocation && global.recordLocation.magicValues
```

Comment:
This won't work in the browser -- not in the sense that the function will return false, but `global` will cause a ReferenceError because it only exists in node.

I'd recommend:

```js
return typeof recordLocation === 'function' && global.recordLocation.magicValues
```

Reply:
Thanks